test/protected_resource_test.rb

```diff
@@ -0,0 +1,101 @@
+require 'test_helper'
+
+class TestProtectedResource < Minitest::Test
+  include CircuitBreakerHelper
+  include ResourceHelper
+
+  def setup
+    Semian.destroy(:testing)
+  rescue
+    nil
+  end
+
+  def teardown
+    destroy_resources
+  end
+
+  def test_acquire_without_bulkhead
+    Semian.register(
+      :testing,
+      exceptions: [SomeError],
+      error_threshold: 2,
+      error_timeout: 5,
+      success_threshold: 1,
+      bulkhead: false,
+    )
+
+    10.times do
+      Thread.new do
+        block_called = false
+        @resource = Semian[:testing]
+        @resource.acquire { block_called = true }
+        assert_equal true, block_called
+        assert_instance_of Semian::CircuitBreaker, @resource.circuit_breaker
+        assert_nil @resource.bulkhead
+      end.join
```

Comment:
This won't do them in parallel, so it won't check the circuit. You'll need to spawn them in parallel, add them to an array, do `threads.each(&:join)`.